components/edit-collective/sections/Policies.js

```diff
@@ -82,6 +82,10 @@ const messages = defineMessages({
     defaultMessage:
       'Only allow expenses to be created by Team Members and Financial Contributors (they may invite expenses from other payees).',
   },
+  'expensePolicy.approversCannotPayExpenses': {
+    id: 'collective.expensePolicy.approversCannotPayExpenses',
```

Comment:
Not a requested change, but just FYI we noticed that IDs were not necessary for `formatjs`. For most new developments we're now omitting it.

```suggestion
```